src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
+@import 'utils/variables';
+@import 'utils/mixin';
+
+html {
+  @include font;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  height: 408px;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin-bottom: 40px;
+
+    .imac {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__description {
+    margin-bottom: 16px;
+
+    .name {
+      @include font;
+
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: #060b35;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    .code {
+      @include font;
+
+      margin-block: 0;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: #616070;
+    }
+  }
+
+  .rating {
+    @include spacebetweenblock;
+
+    margin-bottom: 24px;
+
+    .stars {
+      display: flex;
+
+      & :nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+        margin-right: 4px;
+
+        @include star;
+      }
+
+      &__star:nth-child(5) {
+        background-image: url(../images/star.svg);
+
+        @include star;
+      }
+    }
```

Comment:
Move stars styles to the separate file and use styles for this block from the previous task